curriculum/02-data-wrangling.md

```diff
@@ -7,49 +7,60 @@
 
 ## 1 Curriculum
 
-| SLU   | Name | Specialization | Status   | Content           |
+| BLU   | Name | Specialization | Status   | Content           |
 |-------|------|----------------|----------|-------------------|
-| BLU01 | Advanced wrangling | Data Wrangling | Proposed | [Content](#blu01) |
-| BLU02 | Multi-indexing | Data Wrangling | Proposed | [Content](#blu02) |
+| BLU01 | Messy data | Data Wrangling | Proposed | [Content](#blu01) |
+| BLU02 | Advanced wrangling  | Data Wrangling | Proposed | [Content](#blu02) |
 | BLU03 | Data sources | Data Wrangling | Proposed | [Content](#blu03) |
 
 ## Contents
 
 ### BLU01
 
-#### Advanced wrangling (TBD)
+#### Messy data 
 
 ##### Main topics
 
-1. Transforming datasets 
-2. Merge apply combine 
-3. Transformers and Pipelines 
+1. Alternative tabular sources 
+1. Non tabular data sources 
+1. Common problems and solutions
 
 ##### Detailed curriculum
-[WIP, being developed here](https://docs.google.com/spreadsheets/d/18cNXz5JEgfqG4PpPPVesTJfQs9pMbnMiMiAWCHGm6yc/edit#gid=0)
+- tsv 
+- Unusual delimeters (e.g. ";", "|", etc)
+- excel 
+- corrupt data (e.g. csv files with text boxes over the data) 
+- Using basic unix to inspect files 
+- json data
+- encodings
 
 ### BLU02
-
-#### Multi-indexing
+#### Advanced wrangling (TBD)
 
 ##### Main topics
 
-1. Selecting 
-2. Stacking and unstacking 
-3. Groupby on multi-index
+1. Creating tidy datasets 
+1. Dealing with the index 
+1. Dealing with larger datasets 
+1. Pipelines and Feature Unions 
+1. String methods 
 
 ##### Detailed curriculum
-[WIP, being developed here](https://docs.google.com/spreadsheets/d/18cNXz5JEgfqG4PpPPVesTJfQs9pMbnMiMiAWCHGm6yc/edit#gid=0)
+- Tidy dataset concept 
+- Creating scripts to build tidy datasets
+- Advanced group-apply-combine with lambda functions 
```

Comment:
I'm feeling this one is a bit out of scope here 🤔 
I mean, the applying part I can understand, like applying a "cleaning" function on a dataframe.
But I can't remember a use case for the grouping part.